src/assets/stylesheets/AstroPiModel.scss

```diff
@@ -83,8 +83,8 @@
         flex-flow: column;
         align-items: center;
         text-align: center;
-        padding: $space-0-5;
-        padding-top: $space-0-25;
+        padding-inline: $space-0-5;
+        padding-block-start: $space-0-25;
         box-sizing: border-box;
```

Comment:
Just checking but will `padding: $space-0-5` apply padding to top, right, bottom and left; then `padding-top` adds an extra 0-25 to the top?

so would this need to be:
```
padding-inline: $space-0-5;
padding-block-end: $space-0-5;
padding-block-start: $space-0-75;
```

Reply:
I replaced that with correct values, thank you for spotting it

Reply:
Thanks Magda. Does ` padding-block: $space-0-25 $space-0-5;` need to be ` padding-block: $space-0-75 $space-0-5;`? 

I think the original applies `$space-05` to all sides, then an additional `$space-0-25` to the top

Reply:
I don't know where this `$space-0-75` coming from? The original, as you mentioned, applies `$space-05` to all sides and we have override the top with `$space-0-25`, which is the equivalent of ```padding-top: $space-0-25;
padding-right: $space-0-5;
padding-bottom: $space-0-5;
padding-left: $space-0-5;```
so it translates to `padding-block: $space-0-25 $space-5;` and `padding-inline: $space-0-5;`

Reply:
Ah, perhaps I misunderstood. I didn't realise that `padding-top` overrides the padding: I thought it added to it!